Tests/SnapshotTests/AccountDetailsScreen.swift

```diff
@@ -0,0 +1,61 @@
+import XCTest
+import SnapshotTesting
+import Resolver
+@testable import GoldenAccountsUseCase
+@testable import AccountsJourney
+import AccessControlClient3Gen2
+import ArrangementsClient2Gen2
+import Backbase
+//@testable import GoldenSampleApp
+
+class AccountDetailsScreenSnapshotTests: XCTestCase {
+    override func setUp() {
+        isRecording = true
```

Comment:
i think we can remove this, and the commented out import above

Reply:
Done